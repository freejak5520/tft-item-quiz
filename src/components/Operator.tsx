const Operator = ({ children }: { children?: React.ReactNode }) => (
  <div className="p-4 text-xl text-gray-950 dark:text-gray-50">{children}</div>
);

export default Operator;
