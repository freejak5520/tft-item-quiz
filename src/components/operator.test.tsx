import { render } from "@testing-library/react";
import Operator from "./operator";

describe("Operator", () => {
  it("renders the operator", () => {
    const { container } = render(<Operator />);

    expect(container).toMatchSnapshot();
  });
});
