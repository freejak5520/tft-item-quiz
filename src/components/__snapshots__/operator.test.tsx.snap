// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Operator renders the operator 1`] = `
<div>
  <div
    class="p-4 text-xl text-font-950 dark:text-font-50"
  />
</div>
`;
